### PR TITLE
Remove actor from top panel button, fixes deprecation messages due to…

### LINKED ISSUE #109: Deprecation messages from top panel button
There are repeating deprecation messages from gTile top panel button, from ST Button class, due to deprecation of actor in this class

### PR DESCRIPTION
**Pull request by scherepanov**
… deprecating actor in ST button

Fixes #109

### CODE REVIEW

**Comment by Strauman** [comment]
This doesn't run locally at my computer. Shell version 3.28.4. 

Seems to be a problem in the removal of `actor` in `GTileStausButton` in [extension.js:174](https://github.com/gTile/gTile/pull/110/files#diff-06f1274e40de25abda72d812b1cce86aL172-R174)?

Log from reload:
```
feb. 17 08:40:41 gnome-shell[5285]: gTile Init settings
feb. 17 08:40:41 gnome-shell[5285]: gTile grid-sizes set to 8x2,6x2,3x2
feb. 17 08:40:41 gnome-shell[5285]: gTile Starting...
feb. 17 08:40:41 gnome-shell[5285]: gTile initGrids
feb. 17 08:40:41 gnome-shell[5285]: gTile New Grid for monitor 0
feb. 17 08:40:41 gnome-shell[5285]: Extension "gTile@vibou" had error: TypeError: this.add_style_class_name is not a function
feb. 17 08:40:41 gnome-shell[5285]: JS WARNING: [/usr/share/gnome-shell/extensions/ubuntu-dock@ubuntu.com/appIcons.js 1028]: unreachable code after return statement
```

File: extension.js [hunk 1]
```diff
@@ -169,22 +171,22 @@ const GTileStatusButton = new Lang.Class({
     _init: function(classname) {
         this.parent(0.0, "gTile", false);
 
-        this.actor.add_style_class_name(classname);
+        this.add_style_class_name(classname);
```
Comment (Strauman) [inline extension.js]: This fails for me. See https://github.com/gTile/gTile/pull/110#issuecomment-586858487

Comment (scherepanov) [reply extension.js]: Yes... this is bad problem. On gnome-shell 3.32 this line generate exception with warning about deprecating actor in ST button.
I fixed by removing agent, but had no way to test on earlier versions like 3.28
Obviously, you have a correct fix.
Most likely code needs to discriminate for version of gnome-shell, use actor for version below 3.32 and no actor above.
Anu idea how to get gnome-shell version?

Comment (Strauman) [reply extension.js]: `imports.misc.config.PACKAGE_VERSION`

Comment (scherepanov) [reply extension.js]: Thanks!
Will post fix. I cannot commit from office, can do only over weekend.

Comment (scherepanov) [reply extension.js]: Yes, you can keep actor in this.actor.connect, it is not causing deprecation message.
No need to check for gnome-shell version, it is working on most recent. And, apparently it is also working on previous versions.

**Pull request merged**